databases/relational-databases.md

```diff
@@ -39,69 +69,128 @@ To understand, query, and insert information into a relational database, a techn
 - Users of databases
   - Front end people should have concepts and write very basic SQL
   - Back end people spend a large amount of time with databases
-- Structure of a relational database; tables with columns and references to other tables
+- [Structure of a relational database](https://www.lucidchart.com/pages/database-diagram/database-design)
```

Comment:
Yay!  Now please add this link and the 4 basic SQL verbs - and all links used in the lesson, for that matter - to the ### Materials section.